media_driver/agnostic/gen12/cm/cm_hal_g12.h

```diff
@@ -28,7 +28,7 @@
 #define _CM_HAL_G12_H_
 
 #include "cm_hal.h"
-#include "mhw_render_hwcmd_g8_X.h"
+#include "mhw_render_hwcmd_g12_X.h"
```

Comment:
It seems driver can be successfully built even if this include file be removed from here. Is it actually needed?

Reply:
I'll need to study the actual includes generated by CMake, but I suspect we get "lucky" somehow? Seems like we should include the header since `mhw_render_g12_X` is used in `cm_hal_gen12.cpp`.

Reply:
Whoops, reading too quickly. `mhw_render_g12_X.h` is already included in `cm_hal_gen12.cpp`. We should go with your comment and delete the include from `cm_hal_g12.h`. Thanks.